### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by PayamM1205**

### CODE REVIEW

**Review by Alijocker369 — COMMENTED** [review]
SECURITY.md

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
```
Comment (Alijocker369) [inline SECURITY.md:1]: ****

Comment (Alijocker369) [reply SECURITY.md:1]: SECURITY.md

**Comment by meisamkord1983** [comment]
#84

**Comment by meisamkord1983** [comment]
#84

**Review by naxoootcnjncgjnnxfhjjv — COMMENTED** [review]
SECURITY.md

**Pull request closed without merging**